apps/shell/tash_main.c

```diff
@@ -423,6 +424,13 @@ int tash_execute_cmdline(char *buff)
 					*(buff + 1) = ASCII_NUL;
 					break;
 
+				case ASCII_RSQUOTE:
+					/* Argument is started with opening single quotation mark */
```

Comment:
```suggestion
					/* Argument is starting with opening single quotation mark */
```

Reply:
@Tirtha-Samsung Could you check Abhishek's comment above?